subprojects/file-watching/src/main/java/org/gradle/internal/watch/vfs/impl/WatchingVirtualFileSystem.java

```diff
@@ -73,7 +73,7 @@ public WatchingVirtualFileSystem(
     }
 
     @Override
-    public SnapshotHierarchy getRoot() {
+    protected SnapshotHierarchy getRoot() {
         return rootReference.getRoot();
     }
```

Comment:
Can we move `rootReference` to `AbstractVirtualFileSystem` now?

Reply:
Yes, I moved it.